src/pkgcheck/checks/acct.py

```diff
@@ -35,13 +37,16 @@ def __init__(self, kind, identifier, pkgs):
 
     @property
     def desc(self):
-        return (
-            f"conflicting {self.kind} id {self.identifier} usage: "
-            f"[ {', '.join(self.pkgs)} ]")
+        pkgs = ', '.join(self.pkgs)
+        return f"conflicting {self.kind} id {self.identifier} usage: [ {pkgs} ]"
 
 
 class OutsideRangeAccountIdentifier(results.VersionResult, results.Error):
-    """UID/GID outside allowed allocation range."""
+    """UID/GID outside allowed allocation range.
+
+    To view the range accepted for this repository, look at th file
```

Comment:
```suggestion
    To view the range accepted for this repository, look at the file
```

Reply:
May I just say how much am I thankful for all of those!